services/google-authorization-finder.js

```diff
@@ -5,15 +5,15 @@ var logger = require('./logger');
 var ServiceUrlGetter = require('./service-url-getter');
 var errorMessages = require('../utils/error-messages');
 
-function GoogleAuthorizationFinder(renderingId, accessToken, envSecret) {
+function GoogleAuthorizationFinder(renderingId, forestJwt, envSecret) {
   this.perform = function () {
     return new P(function (resolve, reject) {
       var forestUrl = new ServiceUrlGetter().perform();
 
       request
         .get(forestUrl + '/forest/renderings/' + renderingId + '/google-authorization')
         .set('forest-secret-key', envSecret)
-        .set('google-access-token', accessToken)
+        .set('forest-jwt', forestJwt)
```

Comment:
`forest-token` is a better name. Whatever the technology we use, this is a token.